<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gPotential">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>320</y>
      <width>200</width>
      <height>130</height>
     </rect>
    </property>
    <property name="title">
     <string>Potential</string>
    </property>
    <widget class="QRadioButton" name="bWell">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Potential Well</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="bWS">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Woods Saxon Potential</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="b2RG">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2 Range Gausian</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gE">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>E [MeV]</string>
    </property>
    <widget class="QLineEdit" name="lineReE">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>25</y>
       <width>140</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QLabel" name="lReE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Re</string>
     </property>
    </widget>
    <widget class="QLabel" name="lImE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Im</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineImE">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>65</y>
       <width>140</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gTarget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>120</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>Target Nucleus</string>
    </property>
    <widget class="QLineEdit" name="lineProton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="lProton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>num of proton </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineNeutron">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>65</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="lNeutron">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>num of neutron</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gInitWave">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>10</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>Initial wave packet</string>
    </property>
    <widget class="QLineEdit" name="lineX0">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>-15</string>
     </property>
    </widget>
    <widget class="QLabel" name="lX0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>center of x</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineWaveWidth">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>65</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="lWaveWidth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>width</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gPotentialParam">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>240</y>
      <width>200</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="title">
     <string>Potential Parameter</string>
    </property>
    <widget class="QLineEdit" name="lineR0">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1.27</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineReV">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>65</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>-50</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineImV">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>105</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineA">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>145</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0.67</string>
     </property>
    </widget>
    <widget class="QLabel" name="lR0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>r0 [fm]</string>
     </property>
    </widget>
    <widget class="QLabel" name="lReV">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Re(V0) [MeV]</string>
     </property>
    </widget>
    <widget class="QLabel" name="lImV">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Im(V0) [MeV]</string>
     </property>
    </widget>
    <widget class="QLabel" name="lA">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>a [fm]</string>
     </property>
    </widget>
    <widget class="QLabel" name="lReV1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Re(V1) [MeV]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineReV1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>185</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QLabel" name="lImV1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Im(V1) [MeV]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineImV1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>225</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lMu0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>μ0 [1/fm^2]</string>
     </property>
    </widget>
    <widget class="QLabel" name="lMu1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>μ1 [1/fm^2]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMu0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>265</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0.16</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineMu1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>305</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0.04</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gTime">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>801</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>time = 0</string>
    </property>
    <widget class="QScrollBar" name="barTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>760</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lTmin">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineTmax">
     <property name="geometry">
      <rect>
       <x>682</x>
       <y>50</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lDt">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dt [fm]  = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineDt">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>95</y>
       <width>300</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>5.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gAxis">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>120</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="title">
     <string>Axis</string>
    </property>
    <widget class="QLineEdit" name="lineXmin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>-20</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineXmax">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>65</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineYmin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>105</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>-1.2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineYmax">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>145</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1.2</string>
     </property>
    </widget>
    <widget class="QLabel" name="lXmin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>xmin [fm]</string>
     </property>
    </widget>
    <widget class="QLabel" name="lXmax">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>xmax [fm]</string>
     </property>
    </widget>
    <widget class="QLabel" name="lYmin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ymin [fm]</string>
     </property>
    </widget>
    <widget class="QLabel" name="lYmax">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ymax [fm]</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gInfo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>800</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Infomation</string>
    </property>
    <widget class="QLabel" name="lexpX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>390</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;x&gt; = </string>
     </property>
    </widget>
    <widget class="QLabel" name="lexpPsi">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>390</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;ψ|ψ&gt; = </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="lhc">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>390</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>hc = </string>
     </property>
    </widget>
    <widget class="QLabel" name="lmp">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>390</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>m_p =</string>
     </property>
    </widget>
    <widget class="QLabel" name="lmn">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>90</y>
       <width>390</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>m_n =</string>
     </property>
    </widget>
    <widget class="QLabel" name="lDx">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>120</y>
       <width>390</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dx = </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
